client/src/components/mapping/TabColumnPicking/StaticValueForm/index.tsx

```diff
@@ -71,6 +71,7 @@ const StaticValueForm = ({ attribute }: Props): React.ReactElement => {
       setStaticValue('');
       getFhirTypes();
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What is this?

Reply:
to prevent errors from the linter about the f*cking dependency array on Rect.useEffect(). I don't know since when but now warnings are treated as errors in the CI (yarn does that automatically) so the build couldn't pass otherwise.